Mathlib/Analysis/Convex/Side.lean

```diff
@@ -398,8 +398,12 @@ theorem _root_.Wbtw.wOppSide₁₃ {s : AffineSubspace R P} {x y z : P} (h : Wbt
   rcases ht0.lt_or_eq with (ht0' | rfl); swap
   · rw [lineMap_apply_zero]; simp
   refine' Or.inr (Or.inr ⟨1 - t, t, sub_pos.2 ht1', ht0', _⟩)
-  simp_rw [lineMap_apply, vadd_vsub_assoc, vsub_vadd_eq_vsub_sub, ← neg_vsub_eq_vsub_rev z x,
-    vsub_self, zero_sub, ← neg_one_smul R (z -ᵥ x), ← add_smul, smul_neg, ← neg_smul, smul_smul]
+  rw [lineMap_apply, vadd_vsub_assoc, vsub_vadd_eq_vsub_sub]
+  -- TODO: this was a single `simp_rw` prior to adding the "simp made no progress feature"
+  -- this is worrying!
+  simp_rw [← neg_vsub_eq_vsub_rev z x]
+  rw [vsub_self]
+  simp_rw [zero_sub, ← neg_one_smul R (z -ᵥ x), ← add_smul, smul_neg, ← neg_smul, smul_smul]
```

Comment:
here to, I got this working by adding a `_` after each lemma `simp_rw` wasn't happy about.

Reply:
I had intended to revert these changes... I guess I forgot to push.